jsdoc.conf.json

```diff
@@ -0,0 +1,26 @@
+{
+    "plugins": [],
+    "recurseDepth": 10,
+    "source": {
+        "includePattern": ".+\\.js(doc|x)?$",
+        "excludePattern": "(^|\\/|\\\\)_",
+        "include": [
+            "."
+        ],
+        "exclude": [
+            "node_modules"
+        ]
+    },
+    "sourceType": "module",
+    "tags": {
+        "allowUnknownTags": true,
+        "dictionaries": [
+            "jsdoc",
+            "closure"
+        ]
+    },
+    "templates": {
+        "cleverLinks": false,
+        "monospaceLinks": false
+    }
+}
```

Comment:
Let's use the Postman's jsdoc theme and output options:
```json
"opts": {
    "template": "./node_modules/postman-jsdoc-theme",
    "encoding": "utf8",
    "destination": "./out/docs",
    "recurse": true,
    "readme": "README.md"
  },

  "markdown": {
    "parser": "gfm",
    "hardwrap": false
  }
```

Ref: https://github.com/postmanlabs/newman/blob/develop/.jsdoc-config.json